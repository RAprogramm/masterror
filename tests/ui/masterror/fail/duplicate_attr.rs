use masterror::{AppCode, AppErrorKind, Masterror};

#[derive(Debug, Masterror)]
#[error("dup")]
#[masterror(code = AppCode::Internal, category = AppErrorKind::Internal)]
#[masterror(code = AppCode::Internal, category = AppErrorKind::Internal)]
struct Duplicate;

fn main() {}
