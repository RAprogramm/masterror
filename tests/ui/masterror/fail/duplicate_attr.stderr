error: duplicate #[masterror(...)] attribute
 --> tests/ui/masterror/fail/duplicate_attr.rs:6:1
  |
6 | #[masterror(code = AppCode::Internal, category = AppErrorKind::Internal)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: unused imports: `AppCode` and `AppErrorKind`
 --> tests/ui/masterror/fail/duplicate_attr.rs:1:17
  |
1 | use masterror::{AppCode, AppErrorKind, Masterror};
  |                 ^^^^^^^  ^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
